### ISSUE TITLE
[BUG] ClassicUO -Window Changing keybind issue.  [Intermittent]

### ISSUE THREAD

**Issue by MARPATdroid** [markdwags/Razor#158]
Client: ClassicUO 0.1.7.0
Platform: Windows (two different computers have the same symptoms)
TL;DR: When switching between windows for multiple characters, or to use other applications keybinds for Razor re-assign themselves intermittently.  (IE: pressing enter or space now wants to tame animals which is usually bound to 'g', and Esc wants to guard all, which is bound to 'm'  

Specifics:  This bug seems to be tied to text input or keyboard usage in other windows.  If I move to a window and scroll or click around to read it doesn't usually appear.   However if I start typing and chatting in say discord, or on any other web based chat platform it happens fairly consistently.   

Resolution requires finding the offending script and rebinding it's key (can be an issue if you have multiple similar scripts, IE: tame and wait for target, tame and target last, tame and target by name or if your command is a kill all and you accidentally sic a pack of nightmares you are walking through a dungeon on yourself because you thought you cast greater heal....)
Alternate resolution requires closing the client completely and relaunching/logging in.

Here's a link to a quick demo of the bug: https://youtu.be/cVq8YBn3GOA

**Comment by markdwags** [comment]
This is a very odd one, thanks for the detail.  It sounds more like it might be an issue with ClassicUO and your setup, since ClassicUO shouldn't be picking up your keypresses and sending them to Razor if ClassicUO isn't active.

The only way Razor would be getting key presses is if ClassicUO was sending them to it.

**Comment by MARPATdroid** [comment]
It well could be, I'll copy this over to them as well.  Thanks for the quick response.

**Comment by Wayslayer** [comment]
I have a problem with Razor that's very similar where for some reason every now and then my hotkeys break, and any hotkey not set in razor, and set in UO no longer works, and razor continually trys to use a hotkey from it, for other buttons pressed. If I press the key for that hotkey that its trying to use, it will usually fix it, but sometimes will go to another hotkey, and I have to press the actual hotkey for that. Happens on 2 different clients for 2 shards.  I have them in two totally separated folders, and have reinstalled both clients multiple times.  If i hit my razor hotkey to disable razor hotkeys, then all of my in game macros work just fine and I have no issues.

**Comment by markdwags** [comment]
This issue doesn't appear wide spread, closing for now and is almost a year old now.
